web/public/js/directives/cd-child-card/template.dust

```diff
@@ -60,13 +60,13 @@
             <div ng-class="{'col-md-6': $ctrl.manageDojos, 'col-md-12': !$ctrl.manageDojos}" class="col-xs-12">
               <div ng-show="$ctrl.manageDojos">
                 <h4>{{ 'MEMBER_OF' | translateFromKey }}</h4>
-                <p>{{ 'REMOVE_CHILD_FROM_DOJO_TEXT' | translateFromKey:{name: $ctrl.child.name} }}</p>
+                <p>{{ 'REMOVE_CHILD_FROM_DOJO_TEXT' | translateFromKey:{name: $ctrl.child.firstName || $ctrl.child.name} }}</p>
               </div>
               <cd-picture-grid ng-show="$ctrl.child.dojoListings.length > 0" items="$ctrl.child.dojoListings" on-selection="$ctrl.manageDojos ? $ctrl.removeFromDojo : $ctrl.noop" item-icon="minus" item-icon-color="red" show-item-icons="$ctrl.manageDojos" align="left" size="medium"></cd-picture-grid>
             </div>
             <div class="col-xs-12 col-md-6" ng-show="$ctrl.manageDojos">
               <h4>{{ 'AVAILABLE_DOJOS' | translateFromKey }}</h4>
-              <p ng-bind-html="'ADD_CHILD_TO_DOJO_TEXT' | translateFromKey:{name: $ctrl.child.name, findADojoUrl: '/'}:'escapeParameters'"></p>
+              <p ng-bind-html="'ADD_CHILD_TO_DOJO_TEXT' | translateFromKey:{name: $ctrl.child.firstName || $ctrl.child.name, findADojoUrl: '/'}:'escapeParameters'"></p>
```

Comment:
the `:'escapeParameters'` at the end... escapes the parameters... 😝  So HTML from firstName || name will be escaped.